### ISSUE TITLE
If you set any of the colors on the BODY or A elements you must set all of them.

### ISSUE THREAD

**Issue by lfrancis** [cultureondemand/iava-report#6]
If you set any of the colors on the BODY or A elements you must set all of them.
WCAG 2.0 AA F24 Section 508 (2017) AA F24
1 pages

In HTML the color attributes are TEXT, BGCOLOR, LINK, ALINK and VLINK. In CSS the attributes are COLOR and BACKGROUND-COLOR. Some users have browser defaults set to white text on a black background, so setting one color without setting the others can result in black text on a black background.

https://cultureondemand.github.io/iava-report/assets/css/scrolly.css
Line 1
